ReSharper.FSharp/src/FSharp.Psi.Features/src/Daemon/Tooltips/FSharpIdentifierTooltipProvider.fs

```diff
@@ -27,6 +27,15 @@ type FSharpIdentifierTooltipProvider(lifetime, solution, presenter, xmlDocServic
         (RichText.Empty, text.GetFormattedParts()) ||> Seq.fold (fun result part ->
             result.Append(part.Text.Replace("<", "&lt;").Replace(">", "&gt;").Replace("\n", "<br>"), part.Style))
 
+    //todo: remove explicit formatting with HTML
```

Comment:
The todo should be about moving it to the platform.

@DedSec256 @kirillgla It should be done inside the platform APIs, since these tooltips may be used in different implementations (e.g. tooltips vs quick docs, or Rider vs R#) and the language support should not deal with this kind of low level details, especially due to they tend to change.